content/english/android-tictactoe/puzzle3.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Puzzle 3"
+date: 2020-07-27T14:17:07.42-07:00
+draft: true
+weight: 11
+---
+
+# Puzzle 3a
+This puzzle will test your knowledge about variables. There is something strange with one of the player's color on the board.
+
+Find `// PUZZLE 3a` in the `MainActivity.kt` file and update the variable to show green colored pieces on the board.
```

Comment:
Will the PUZZLEs in the code contain sufficient explanation as to what to do? These explanations for the tasks are a little vague.